packages/cli/src/commands/init.ts

```diff
@@ -192,26 +193,20 @@ export default class Init extends Command {
   }
 
   async setupProject(flags: any): Promise<void> {
-    const [
-      defaultSpecVersion,
-      defaultRepository,
-      defaultEndpoint,
-      defaultAuthor,
-      defaultVersion,
-      defaultDescription,
-      defaultLicense,
-    ] = await readDefaults(this.projectPath);
-
-    // Should use template specVersion as default, otherwise use user provided
-    flags.specVersion = defaultSpecVersion ?? flags.specVersion;
+    const [defaultEndpoint, defaultAuthor, defaultDescription] = await readDefaults(this.projectPath);
 
-    this.project.endpoint = await cli.prompt('RPC endpoint:', {
-      default: defaultEndpoint ?? 'wss://polkadot.api.onfinality.io/public-ws',
-      required: true,
+    this.project.endpoint = !Array.isArray(defaultEndpoint) ? [defaultEndpoint] : defaultEndpoint;
+    const userInput = await cli.prompt('RPC endpoint:', {
+      default: defaultEndpoint[0] ?? 'wss://polkadot.api.onfinality.io/public-ws',
+      required: false,
     });
-
-    this.project.repository = await cli.prompt('Git repository', {required: false, default: defaultRepository});
-
+    // Should assume all project endpoints will be string[]
+    if (!Array.isArray(this.project.endpoint)) {
```

Comment:
Line 198 converts it to an array, I think this is redundant. Then the following if will always be false.